src/Hintman/Config.hs

```diff
@@ -44,9 +44,11 @@ hintmanConfiguationCodec = HintmanConfig
 
 loadFileConfig :: FilePath -> IO HintmanConfig
 loadFileConfig path =
-    Toml.decodeFile hintmanConfiguationCodec path `catch` catchLoad
+    (Toml.decodeFile hintmanConfiguationCodec path `catch` catchExcAndReturnDefaultConfig)
+      `catch` catchExcAndReturnDefaultConfig
   where
-    catchLoad :: LoadTomlException -> IO HintmanConfig
-    catchLoad _ = do
-        print $ "Hintman failed to load config from " <> path
-        pure defaultHintmanConfig
+    catchExcAndReturnDefaultConfig :: SomeException -> IO HintmanConfig
+    catchExcAndReturnDefaultConfig e = case e of
+        Exc (_ :: LoadTomlException) -> defaultHintmanConfig <$ print ("Encountered LoadTomlException while trying to load config from " <> path)
+        Exc (_ :: IOException) -> defaultHintmanConfig <$ print ("Config '" <> path <> "' does not exist! Using default configuration")
```

Comment:
For outputting text to terminal it's better to use `putTextLn` function from `relude`, not `print`. `print` will add extra quotes

Reply:
Noted, now the test assertions look nicer. But with `toText (show e)` I get a type checking error that tells me I need to add an annotation... slotting in `@String` work but is that the best way?

Reply:
@qoelet , `show` is polymorphic in `relude`. So when you use `putTextLn . show` it understands that the type is `Text` and you don't need to do anything else. But when you say `putTextLn . toText . show` he thinks so: Okay, for `putTextLn` I need `Text`; function `toText` gives me `Text` from polymorphic type `a`, and function `show` gives you this type `a` (because it is polimorphic) from the given `Exception` (or whatever). So how would it know what is this `a` should be? It could be anything.
Hope it makes the error clearer.

Reply:
Yes it's clearer to me now, fixing the PR in a bit, thanks alot @vrom911, @ChShersh ... I'm definitely learning with this PR :)